### PR TITLE
Add instructions for remoting when using XR SDK

### PR DESCRIPTION
**Pull request by MaxWang-MS**
## Overview
This PR improves the remoting documentation by including separate paths for people to enable remoting based on whether legacy XR or XR SDK is used.

## Changes
- Fixes: #8716

### CODE REVIEW

**Comment by davidkline-ms** [comment]
@MaxWang-MS, the documentation generator found an invalid link

```
(../../Documentation/ReleaseNotes.md#L124)Invalid link: '<a href="../../Documentation/Tools/HolographicRemoting.md#hololens-2">Holographic Remoting</a>'. The file ../../Documentation/Tools/HolographicRemoting.md doesn't contain a bookmark named 'hololens-2'.
```

**Comment by davidkline-ms** [comment]
The link (in ReleaseNotes.md, line 124) was broken by changing 

```
### HoloLens 2
```

to

```
### Legacy XR setup instructions (only needed by HoloLens 2)
```

**Review by davidkline-ms — APPROVED** [review]

File: Documentation/Tools/HolographicRemoting.md [hunk 1]
```diff
@@ -115,21 +111,29 @@ You can also temporarily remove the adapter to workaround your issue via the fol
 1. (Optional) Navigate to MRTK / Providers / WindowsMixedReality / Shared / DotNetAdapter in your file explorer (not Unity's Assets view) and delete the `.bin` and `.obj` folders. This removes the local cache of NuGet restored packages for DotNetWinRT.
 1. If you run the MRTK Configurator again, make sure you don't re-enable MSBuild for Unity.
 
+### XR SDK setup instructions
+
+1. Open **Window > Package Manager**
+    - Verify that latest version of the **Windows Mixed Reality** package is installed.
```
Comment (keveleigh) [inline Documentation/Tools/HolographicRemoting.md:117]: The XR SDK package has a slightly different name: "Windows XR Plugin"

File: Documentation/Tools/HolographicRemoting.md [hunk 2]
```diff
@@ -115,21 +111,29 @@ You can also temporarily remove the adapter to workaround your issue via the fol
...
+1. Go to the Standalone settings and ensure Windows Mixed Reality is checked under Plug-in Providers. Also ensure that Initialize XR on Startup is checked.
+1. (**_Optional_**) Click on the Windows Mixed Reality tab under XR Plug-in Management and create a custom settings profile to change the defaults. If the list of settings are already there, no profile needs to be created.
+1. Select **Mixed Reality Toolkit** > **Utilities** > **Windows Mixed Reality** > **Check Configuration** to make sure the correct scripting define is set.
```
Comment (keveleigh) [inline Documentation/Tools/HolographicRemoting.md:120]: We might want to make it clear that DotNetWinRT isn't required for HL2 features on XR SDK. I'm not sure this step is needed

File: Documentation/Tools/HolographicRemoting.md [hunk 3]
```diff
@@ -115,21 +111,29 @@ You can also temporarily remove the adapter to workaround your issue via the fol
 1. (Optional) Navigate to MRTK / Providers / WindowsMixedReality / Shared / DotNetAdapter in your file explorer (not Unity's Assets view) and delete the `.bin` and `.obj` folders. This removes the local cache of NuGet restored packages for DotNetWinRT.
 1. If you run the MRTK Configurator again, make sure you don't re-enable MSBuild for Unity.
 
+### XR SDK setup instructions
```
Comment (keveleigh) [inline Documentation/Tools/HolographicRemoting.md:114]: Do we want to move this next to the legacy header? or maybe decrement the level of the above couple headers to make it clear that they correspond to the legacy path?

**Pull request merged**